what-are-bots.md

```diff
@@ -0,0 +1,20 @@
+### WHAT ARE BOTS?
+
+Bots are also known as Internet Robots, and just as the name Robot implies; Bots make things easier. They can handle repetitive tasks and much more. [Formerly, the popular version of bots came in the form of malware,](https://www.abusix.com/blog/a-brief-history-of-bots-and-how-theyve-shaped-the-internet-today) but that has begun to change with tech companies like Facebook, Telegram, Slack providing a more useful version of this technology, and they represent CHAT BOTS.
+
+##### WHY ARE THEY SO IMPORTANT?
+
+Now let me tell you a story about ChatBots and how they came to be in this ever growing internet generation, evolving gradually to this stage, gathering momentum as it approaches this point.
+
+In the beginning was Mr.Internet, the current King of transmitting information, who ruled with greatness, knowledge, and love, and basically has a lot of tricks under his sleeve. One day, he realized an organizational problem in his kingdom, and since he is a big fan of delegating and organizing, he appoints worthy Knights to handle this problem, Knighthood meaning Websites. So this was where Facebook, Google, Twitter and the rest came in.
+
+Now this improved the situation in the Kingdom, and things were awesome and his Kingdom (information) grew larger. As it grew, things became disorganized once more, and so Mr.Internet called for a meeting, discussed the problem and came up with a solution. He said Why don't you have your own specific kingdoms and territories and mansions where you can receive your guests and they all agreed to this, and then he gave them a condition saying, "_But you must call these territories, **MOBILE APPS.**_"And they were all cool with it and new knights joined their crusade (Whatsapp, SnapChat, and the rest), Now things became cooler and everyone was happy.
+
+One day, the King took a trip and when he finally got back, he noticed a new disorganization problem, this time there were so many territories, and mansions scattered all over the place, relevant and irrelevant. He was not happy with this. And so he called a meeting and stated this problem saying "_Why is every Knight building a mansion or territory for every single purpose when he can just have a room for it? In fact, why can't it be like a mall where you can find all you need? If a person visits Mansion A to buy cereal, why go to another Mansion B to buy Milk. Why can't Mansion B simply sell his Milk at Mansion A? You don't have to travel all around my kingdom to get what you want_"- he said. And that was how **BOTS were formed** And so Facebook, a great knight whom is popular for having many visitors offered to share visitors with any Knight who came to hang out in their mansion (messenger platform). And by visitors, I mean about **1 billion users a day**.
+
+##### THIS IS WHY CHATBOTS ARE IDEAL FOR YOUR IDEA AND BUSINESS.
```

Comment:
What do you think of replacing this with something that links the section below to the story above? Something like:
To summarize;
* website ...